### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by rongncao**
you need change shhs_base_dir to your local drive
also this code uses utils and plot we developed for hw5

### CODE REVIEW

File: SHHS model - RNN LSTM RF.py [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+test_dataset = load_dataset(X_test_new,y_test, MODEL_TYPE)
+
+train_loader = torch.utils.data.DataLoader(train_dataset, batch_size=BATCH_SIZE, shuffle=True, num_workers=NUM_WORKERS)
+valid_loader = torch.utils.data.DataLoader(valid_dataset, batch_size=BATCH_SIZE, shuffle=False, num_workers=NUM_WORKERS)
+test_loader = torch.utils.data.DataLoader(test_dataset, batch_size=BATCH_SIZE, shuffle=False, num_workers=NUM_WORKERS)
+'''
+
+PATH_OUTPUT = "../output/sleep/"
+os.makedirs(PATH_OUTPUT, exist_ok=True)
+class MyRNN(nn.Module):
```
Comment (dkapp7) [inline SHHS model - RNN LSTM RF.py:291]: is this RNN optimized?

**Review by dkapp7 — DISMISSED** [review]

**Comment by macnb9** [comment]
@rongncao can you move the added files into an existing or new directory to keep them organized by purpose within the project structure?

**Review by macnb9 — CHANGES_REQUESTED** [review]
Repeat of my question to more explicitly block: can you move the added files into an existing or new directory to keep them organized by purpose within the project structure?

**Comment by rongncao** [comment]
will do

**Pull request closed without merging**